SasquatchMac/SasquatchMac/ViewControllers/EventFilterViewController.swift

```diff
@@ -6,16 +6,20 @@ import Cocoa
 class EventFilterViewController: NSViewController {
 
   var appCenter: AppCenterDelegate = AppCenterProvider.shared().appCenter!
+  private var eventFilterStarted = false
 
   @IBOutlet weak var setEnabledButton: NSButton!
 
   @IBAction func setEnabled(_ sender: NSButton) {
+    if !eventFilterStarted {
```

Comment:
What would be if the state is being changed to disabled? Would we still start it?

Reply:
I mean if it is already enabled on the first launch

Reply:
There is no way to start in on the launch, it's intended to start it manually here